clients/unshaded/src/main/java/tachyon/hadoop/AbstractTFS.java

```diff
@@ -58,13 +67,15 @@
   public static final String RECOMPUTE_PATH = "tachyon_recompute/";
 
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
+  // Always tell Hadoop that we have 3x replication.
+  private static final int BLOCK_REPLICATION_CONSTANT = 3;
```

Comment:
I think this is how many replicas to tell HDFS that we currently have, not how many we want to have. The functionality hasn't changed, we're just using a constant now instead of the previous
```
  public int getDiskReplication() {
    // TODO(hy): Implement it.
    return 3;
  }
```
We can decide how to manage block replication info separately from this commit.